test/testcapabilities.cpp

```diff
@@ -7,6 +7,40 @@ class TestCapabilities : public QObject
     Q_OBJECT
 
 private slots:
+    void testPushNotificationsAvailable_pushNotificationsForActivitiesAvailable_returnTrue()
+    {
+        QStringList typeList;
+        typeList.append("activities");
+
+        QVariantMap notifyPushMap;
+        notifyPushMap["type"] = typeList;
+
+        QVariantMap capabilitiesMap;
+        capabilitiesMap["notify_push"] = notifyPushMap;
+
+        const auto &capabilities = OCC::Capabilities(capabilitiesMap);
+        const auto filesPushNotificationsAvailable = capabilities.availablePushNotifications().testFlag(OCC::PushNotificationType::Activities);
+
+        QCOMPARE(filesPushNotificationsAvailable, true);
+    }
+
+    void testPushNotificationsAvailable_pushNotificationsForActivitiesNotAvailable_returnFalse()
+    {
+        QStringList typeList;
+        typeList.append("noactivities");
+
+        QVariantMap notifyPushMap;
+        notifyPushMap["type"] = typeList;
+
+        QVariantMap capabilitiesMap;
+        capabilitiesMap["notify_push"] = notifyPushMap;
+
+        const auto &capabilities = OCC::Capabilities(capabilitiesMap);
```

Comment:
Because we talked about it last week. Why not `const OCC::Capabilities capabilities()`. Just curious.

Reply:
@FlexW Just a copy-paste.  If I use  `const OCC::Capabilities capabilities()` - then will have to change it in all other tests :)

@er-vin What do you think? This sounds like a candidate for a separate PR -> replace all `const auto &capabilities = OCC::Capabilities(capabilitiesMap);` with `const auto capabilities = OCC::Capabilities(capabilitiesMap);`

Reply:
Separate PR is fine by me.